hack/deploy-hub-local/create-vm-sno.yml

```diff
@@ -1,6 +1,7 @@
 parameters:
   nodes: 1
   clusters: 2
+  singlenic: true
```

Comment:
Are we going to always use singlenic when doing SNO?

Reply:
that's the idea...Talked with Flavio, the default will be one nic